XMLProjectIS/MovieNames.cs

```diff
@@ -53,94 +53,102 @@ internal class GenreConverter : JsonConverter
+                    case "Drama":
+                        return Genre.Drama;
+                    case "Erotic":
+                        return Genre.Erotic;
+                    case "Family":
+                        return Genre.Family;
+                    case "Fantasy":
+                        return Genre.Fantasy;
+                    case "Found Footage":
+                        return Genre.FoundFootage;
+                    case "Historical":
+                        return Genre.Historical;
+                    case "Horror":
+                        return Genre.Horror;
+                    case "Independent":
+                        return Genre.Independent;
+                    case "Legal":
+                        return Genre.Legal;
+                    case "Live Action":
+                        return Genre.LiveAction;
+                    case "Martial Arts":
+                        return Genre.MartialArts;
+                    case "Musical":
+                        return Genre.Musical;
+                    case "Mystery":
+                        return Genre.Mystery;
+                    case "Noir":
+                        return Genre.Noir;
+                    case "Performance":
+                        return Genre.Performance;
+                    case "Political":
+                        return Genre.Political;
+                    case "Romance":
+                        return Genre.Romance;
+                    case "Satire":
+                        return Genre.Satire;
+                    case "Science Fiction":
+                        return Genre.ScienceFiction;
+                    case "Short":
+                        return Genre.Short;
+                    case "Silent":
+                        return Genre.Silent;
+                    case "Slasher":
+                        return Genre.Slasher;
+                    case "Sport":
+                        return Genre.Sport;
+                    case "Sports":
+                        return Genre.Sports;
+                    case "Spy":
+                        return Genre.Spy;
+                    case "Superhero":
+                        return Genre.Superhero;
+                    case "Supernatural":
+                        return Genre.Supernatural;
+                    case "Suspense":
+                        return Genre.Suspense;
+                    case "Teen":
+                        return Genre.Teen;
+                    case "Thriller":
+                        return Genre.Thriller;
+                    case "War":
+                        return Genre.War;
+                    case "Western":
+                        return Genre.Western;
+                }
             }
-            throw new Exception("Cannot unmarshal type Genre");
+            catch (Exception ex)
+            {
+
+                Console.WriteLine("Cannot unmarshal type Genre");
+            }
+
```

Comment:
Logging is a good idea, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute after the exception occurs, as if nothing went wrong.  That could be worse than the default behavior: terminate the method when an exception occurs.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().